src/addRemove.js

```diff
@@ -2,108 +2,105 @@
 import './style.css';
 import statusUpdate from './server.js';
 
-    let size = 0;
-    const ul = document.getElementById('task-list');
-    let savedTasks = JSON.parse(localStorage.getItem('savedTasks')) || [];
+let size = 0;
+const ul = document.getElementById('task-list');
+let savedTasks = JSON.parse(localStorage.getItem('savedTasks')) || [];
 
-  const clearContent = () => {
-    savedTasks = savedTasks.filter((object) => object.completed !== true);
-    localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
-    displayAllTask();
-  }
-
-  const addIndex = () => {
-    if (savedTasks.length < 1) {
-      return size;
-    }
-    return savedTasks[savedTasks.length - 1].index + 1;
+const addIndex = () => {
+  if (savedTasks.length < 1) {
+    return size;
   }
-
-  const addTask = (book) => {
-    savedTasks.push(book);
+  return savedTasks[savedTasks.length - 1].index + 1;
+};
+
+const addTask = (book) => {
+  savedTasks.push(book);
+  localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
+  size += 1;
+};
+
+const updateList = () => {
+  let newIndex = 0;
+  savedTasks = JSON.parse(localStorage.getItem('savedTasks')) || [];
+  // eslint-disable-next-line no-return-assign
+  savedTasks.filter((obj) => (obj.index = newIndex++));
+  localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
+};
+
+const editTask = (e, i) => {
+  if (e.key === 'Enter') {
+    savedTasks[i].description = e.target.innerHTML;
     localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
-    size += 1;
+    e.preventDefault();
   }
-
-  const updateList = () => {
-    let newIndex = 0;
-    savedTasks = JSON.parse(localStorage.getItem('savedTasks')) || [];
-    // eslint-disable-next-line no-return-assign
-    savedTasks.filter((obj) => (obj.index = newIndex++));
-    localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
-  }
-
-  const editTask = (e, i) => {
-    if (e.key === 'Enter') {
-      savedTasks[i].description = e.target.innerHTML;
-      localStorage.setItem('savedTasks', JSON.stringify(savedTasks));
-      displayAllTask();
-      e.preventDefault();
+  e.target.parentNode.children[1].contentEditable = true;
+};
+
+// to Delete task from the list and
+// updating the local storage at the same time
```

Comment:
Remove the comments in order to keep them dry.

Reply:
ok sure i will do just that